xml/schema/Event/Participant.xml

```diff
@@ -338,4 +338,22 @@
     <key>id</key>
     <onDelete>SET NULL</onDelete>
   </foreignKey>
+  <field>
+    <name>created_id</name>
+    <type>int unsigned</type>
+    <title>Contact responsible for registering this participant</title>
```

Comment:
```suggestion
    <title>Created by Contact ID</title>
```

Reply:
@mattwire did you see this suggestion? Instead of the change you just pushed to the token label, this title needs to be corrected. Right now you have the comment here instead of the title. The comment is "Contact responsible for registering this participant". The title is supposed to be "Created by Contact ID"